.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,5 +4,6 @@ In order to help your fellow volunteers, please note the following suggestions.
 
 - [ ] Please read and apply the [Pull Request Guidelines](https://github.com/exercism/docs/blob/master/contributing-to-language-tracks/pull-request-guidelines.md).
 - [ ] Please check the [contributing guide's](https://github.com/exercism/problem-specifications/blob/master/.github/CONTRIBUTING.md#contributing) `Table of Contents` and read the relevant sections.
+- [ ] Lastly, in case this PR modifies `canonical-data.json`, please increment its version as [detailed here](https://github.com/exercism/problem-specifications#test-data-versioning). You can also find more general thoughts on exercise versioning [here](https://github.com/exercism/problem-specifications/blob/master/.github/CONTRIBUTING.md#exercise-versioning).
```

Comment:
> You can also find more general thoughts on exercise versioning [here](https://github.com/exercism/problem-specifications/blob/master/.github/CONTRIBUTING.md#exercise-versioning).

The linked section references track-specific versioning (https://github.com/exercism/discussions/issues/158), not problem-specifications versioning. I don't think the link belongs at all.